doc/data_formats.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
+
+.. _data-formats:
+
+Supported data formats
+======================
+
+When MNE-Python loads sensor data, the data are stored in a Python object of
+type :class:`mne.io.Raw`. Specialized loading functions are provided for the
+raw data file formats from a variety of equipment manufacturers. All raw data
+input/output functions in MNE-Python are found in :mod:`mne.io` and start
+with :samp:`read_raw_{*}`; see the documentation for each reader function for
+more info on reading specific file types.
+
+As seen in the table below, there are also a few formats defined by other
+neuroimaging analysis software packages that are supported (EEGLAB,
+FieldTrip). Like the equipment-specific loading functions, these will also
+return an object of class :class:`~mne.io.Raw`.
```

Comment:
we also have some functions to read epochs -- e.g., read_epochs_eeglab. I wonder if that should also be listed here. Also, we can read and write hdf5 files which can be useful for data exchange and interoperability with other software packages. Perhaps worth mentioning somewhere?